README.md

```diff
@@ -1,12 +1,39 @@
 # registry
-Devfile registry - storing contents of the common devfile registry that feeds into the OCI based common registry
+Devfile registry - storing contents of the public devfile registry hosted at https://registry.devfile.io.
```

Comment:
1. change "public" to "public community"
2. I think having the OCI based registry is a good piece of info.

Reply:
Agreed on both points. Updated.